src/components/ha-camera-stream.ts

```diff
@@ -163,6 +184,26 @@ class HaCameraStream extends LitElement {
     }
   }
 
+  private async _renderHLSExoPlayer(url: string) {
+    this._resizeExoPlayerListener=() => this._resizeExoPlayer();
```

Comment:
```suggestion
    this._resizeExoPlayerListener = () => this._resizeExoPlayer();
```
Do you have prettier installed and setup?

Reply:
No, sorry...this was my first time playing with JS/Typescript/Node (and Kotlin on the Android side) so I don't have any dev environments set up.
Can you point me to a tutorial on how to set up a JS/Typescript dev environment?

Reply:
Use VScode. Install the recommended plugins in the .vscode folder. Configure prettier to be your default formatter and then turn format on save (setting in vscode)